src/index.js

```diff
@@ -3,10 +3,14 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import { BrowserRouter as Router } from 'react-router-dom';
+import { store } from './utils/redux/store';
+import { Provider as ReduxProvider } from 'react-redux';
```

Comment:
Just wondering because I am still not 100% on redux myself, but was there a reason why you couldn't just put `import { Provider } from 'react-redux'`

Reply:
Maybe it's not such a problem here, but when there are multiple 'providers' wrapping the app, it can get confusing which provider is providing what. So I decided to rename it here so it's easy to see at a glance what it's doing.

Reply:
Thanks for clarifying! Makes complete sense.